### PR TITLE
Added info regarding the name tag

### PR DESCRIPTION
**Pull request by toshiharutf**
Added info regarding the default name tag in Accelerometer.md.
To be evaluated as a template for other sensors nodes.

Relates to the discussion https://github.com/cyberbotics/webots/discussions/5449

Preview: https://www.cyberbotics.com/doc/reference/accelerometer?version=toshiharutf-doc-acccelerometer-1

### CODE REVIEW

File: docs/reference/accelerometer.md [hunk 1]
```diff
@@ -2,24 +2,28 @@
...
+The [Accelerometer](#accelerometer) node can be used to model accelerometer devices such as those commonly found in mobile electronics, robots and game input devices.
+The [Accelerometer](#accelerometer) node measures acceleration and gravity induced reaction forces over 1, 2 or 3 axes.
+It can be used for example to detect fall, the up/down direction, etc.
+The parent node of an [Accelerometer](#accelerometer) node should have a [Physics](physics.md) node defined in its `physics` field, so that correct measurements can be performed.
+
+### Field Summary
+
 ```
 Accelerometer {
+  SFString name      "accelerometer"  # used by wb_robot_get_device()
```
Comment (omichel) [inline docs/reference/accelerometer.md:16]: Missing space here:
```suggestion
  SFString name       "accelerometer"  # used by wb_robot_get_device()
```

File: docs/reference/accelerometer.md [hunk 2]
```diff
@@ -2,24 +2,28 @@
...
   MFVec3f lookupTable [ ]    # lookup table
   SFBool  xAxis       TRUE   # {TRUE, FALSE}
   SFBool  yAxis       TRUE   # {TRUE, FALSE}
   SFBool  zAxis       TRUE   # {TRUE, FALSE}
```
Comment (omichel) [inline docs/reference/accelerometer.md:20]: Alignment of comments:
```suggestion
  MFVec3f lookupTable [ ]             # lookup table
  SFBool  xAxis       TRUE            # {TRUE, FALSE}
  SFBool  yAxis       TRUE            # {TRUE, FALSE}
  SFBool  zAxis       TRUE            # {TRUE, FALSE}
```

File: docs/reference/accelerometer.md [hunk 3]
```diff
@@ -2,24 +2,28 @@
...
-  SFFloat resolution  -1     # [0, inf)
+  SFFloat resolution  -1     # [0, inf)
```
Comment (omichel) [inline docs/reference/accelerometer.md:21]:
```suggestion
  SFFloat resolution  -1              # [0, inf)
```

File: docs/reference/accelerometer.md [hunk 4]
```diff
@@ -2,24 +2,28 @@
...
+  SFString name      "accelerometer"  # used by wb_robot_get_device()
   MFVec3f lookupTable [ ]    # lookup table
   SFBool  xAxis       TRUE   # {TRUE, FALSE}
   SFBool  yAxis       TRUE   # {TRUE, FALSE}
   SFBool  zAxis       TRUE   # {TRUE, FALSE}
-  SFFloat resolution  -1     # [0, inf)
+  SFFloat resolution  -1     # [0, inf)  
 }
 ```
+**Note**: The description above is a just a partial list of the fields in Accelerometer. The complete list can be found at: [Accelerometer.wrl](https://github.com/cyberbotics/webots/blob/master/resources/nodes/Accelerometer.wrl).
```
Comment (omichel) [inline docs/reference/accelerometer.md:24]:
```suggestion
**Note**: The above description lists only the fields specific to the Accelerometer node. The complete field list can be found in the [Accelerometer.wrl](https://github.com/cyberbotics/webots/blob/master/resources/nodes/Accelerometer.wrl) definition file.
```

File: docs/reference/accelerometer.md [hunk 5]
```diff
@@ -2,24 +2,28 @@
...
 
-### Description
-
-The [Accelerometer](#accelerometer) node can be used to model accelerometer devices such as those commonly found in mobile electronics, robots and game input devices.
-The [Accelerometer](#accelerometer) node measures acceleration and gravity induced reaction forces over 1, 2 or 3 axes.
-It can be used for example to detect fall, the up/down direction, etc.
-The parent node of an [Accelerometer](#accelerometer) node should have a [Physics](physics.md) node defined in its `physics` field, so that correct measurements can be performed.
-
-### Field Summary
+- `name`: This optional field defines the string name used by `wb_robot_get_device()` to get the `WbDeviceTag` to reference this sensor. If not defined in the `.proto` file, the default value described in the `.wrl` file from above is used.
```
Comment (omichel) [inline docs/reference/accelerometer.md:26]:
```suggestion
- `name`: This field defines the string name used by `wb_robot_get_device()` to get the `WbDeviceTag` handle of this sensor. Its default value is "accelerometer".
```

**Review by omichel — CHANGES_REQUESTED** [review]
Thank you for this contribution.
It is a very good idea to move the description above the field summary.
I have a couple of suggestions to further improve it.

**Comment by toshiharutf** [comment]
> Thank you for this contribution. It is a very good idea to move the description above the field summary. I have a couple of suggestions to further improve it.

Thanks for the review. I updated the PR.

**Review by toshiharutf — COMMENTED** [review]
Committed suggested changes

File: docs/reference/accelerometer.md [hunk 6]
```diff
@@ -21,6 +11,20 @@ The parent node of an [Accelerometer](#accelerometer) node should have a [Physic
...
+Accelerometer {
+  SFString name       "accelerometer" # used by wb_robot_get_device()
+  MFVec3f lookupTable [ ]             # lookup table
+  SFBool  xAxis       TRUE            # {TRUE, FALSE}
+  SFBool  yAxis       TRUE            # {TRUE, FALSE}
+  SFBool  zAxis       TRUE            # {TRUE, FALSE}
+  SFFloat resolution  -1              # [0, inf)
+}
+```
+**Note**: The above description lists only the fields specific to the Accelerometer node. The complete field list can be found in the [Accelerometer.wrl](https://github.com/cyberbotics/webots/blob/master/resources/nodes/Accelerometer.wrl) definition file.
```
Comment (omichel) [inline docs/reference/accelerometer.md:24]: Sorry, I missed this one in my first review. This is a better way to link to the correct version of the source code:
```suggestion
**Note**: The above description lists only the fields specific to the Accelerometer node. The complete field list can be found in the [Accelerometer.wrl]({{ url.github_tree }}/resources/nodes/Accelerometer.wrl) definition file.
```

**Review by omichel — CHANGES_REQUESTED** [review]

File: docs/reference/accelerometer.md (hunk 6 above)
Comment (toshiharutf) [reply docs/reference/accelerometer.md:24]: Nice, committing it

**Review by omichel — APPROVED** [review]
Thank you!

**Pull request merged**